### ISSUE TITLE
Incorrect peer dependency warning about React

### ISSUE THREAD

**Issue by zoul** [sto3psl/react-datawrapper-chart#7]
First, thank you very much for this useful package! I am using it with React 18, everything seems to work as intended, but I get the following warning when installing deps:

```
warning " > react-datawrapper-chart@1.1.1" has incorrect peer dependency "react@^16.8".
```

Is it possible to describe the dependency in a way that doesn’t produce the warning? I don’t know much about the version syntax, can we say something like “React 16/17/18 is fine”?

**Comment by MatchuPitchu** [comment]
@zoul I had the same issue, so I updated the `package.json` in the npm package and it worked for me. The package still works, but now React versions up to 18 are also allowed.

```json
"peerDependencies": {
  "react": "^16.8 - ^18"
},
```

@sto3psl I forked your repo and created a pull request: https://github.com/sto3psl/react-datawrapper-chart/pull/8

**Comment by sto3psl** [comment]
Sorry for the delay, I'm not working at Datawrapper anymore and don't use this package at all. 

I just published version 1.1.2 with your changes @MatchuPitchu in #8. If any one of you would like to take over the package, let me know!

**Comment by zoul** [comment]
Thank you! Would it be possible for Datawrapper the company to take over the package? It’s a practical thing to use, React is very common and maintaining it shouldn’t be too time consuming.

**Comment by MatchuPitchu** [comment]
> Sorry for the delay, I'm not working at Datawrapper anymore and don't use this package at all.
> 
> I just published version 1.1.2 with your changes @MatchuPitchu in #8. If any one of you would like to take over the package, let me know!

Thank you @sto3psl . I‘d like to take over this little package, rewrite it with TypeScript and update to React 18 with useSyncExternalStorage. But how does the transfer work?  I haven't been a maintainer of a package yet.

**Comment by sto3psl** [comment]
I created #9 to figure out what happens to this package.
